### DISCUSSION TITLE
Project

### DISCUSSION THREAD

**Discussion by oba4me** [scenttz/Capstone-Project-Diagnostics-fecal-chicken-app-by-deep-learning#1] [General]
Hello scenttz,

please i have similar project i am working on for theses and I need your guidance. Basically need to create an app for poultry disease detector using machine learning.

**Comment by scenttz** [comment]
I use the streamlit website to deploy my model to the app. You can see my code in this [link.](https://github.com/scenttz/Capstone-Project-Diagnostics-fecal-chicken-app-by-deep-learning/blob/main/app.py) and can access to try my app [here](https://scenttz-scenttz-capstone-project-diagnostics-fecal-c-app-67tcm7.streamlit.app/). However, because this project is only for submitting work for the course, it may not be very good. But if you have a question, I'd be happy to give you some advice.

If you want to learn more, check out this [link](https://analyticsindiamag.com/deploy-your-deep-learning-based-image-classification-model-with-streamlit/).

**Reply by oba4me** [reply]
Thank you scenttz, I will look at your code again and see how i can adopt some part, this is also for project defence and but will like to keep it for use for local poultry farmers. 
What will you recommend?

**Reply by oba4me** [reply]
https://scenttz-scenttz-capstone-project-diagnostics-fecal-c-app-67tcm7.streamlit.app/ not opening currently

**Reply by scenttz** [reply]
I've not yet tried my app with actual local poultry farmers but I think It should work. I fix some code of my app, you can try my app again because The Keras API is updated to edit some code at load_model.

**Reply by oba4me** [reply]
Thank you, let me see if i can build anything around this, what did you use for training the model. check images below none gave me up to 50% accuracy
![image_Page_030_Image_0001](https://user-images.githubusercontent.com/22814913/206213119-44caa3d3-03ee-43ab-b916-1d1b26864a07.jpg)
![images](https://user-images.githubusercontent.com/22814913/206213132-09a1accc-e4fc-4f37-9838-70c5ceec1add.jpg)
![new-castle-disease-1](https://user-images.githubusercontent.com/22814913/206213141-09fea2c8-f249-42b4-9072-f1932c99b12b.jpg)

**Reply by scenttz** [reply]
In my opinion, CNN model deep learning use input from pixel color of image. Due to your pictures don't same pixels of color and difference of each image. For improve score of model should try objective detection image before classification image to help model select only characters for classify diseases

**Reply by oba4me** [reply]
Hello Scenttz, i was wondering if i can get what your algorithm looks like to achieve this. Thank you
